### PR TITLE
Added locale id test

### PR DESCRIPTION
**Pull request by manwar**
Hi Team,

Please review PR.

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

**Review by preaction — APPROVED** [review]
Looks good to me

File: t/05register.t [hunk 1]
```diff
@@ -43,6 +43,18 @@ use DateTime::Locale;
     sub short_date_format {'test test2'}
 }
 
+like(
+    exception { DateTime::Locale->register( id => "@" ) },
```
Comment (autarch) [inline t/05register.t]: As I mentioned, I'm a bit OCD about code. I prefer single quotes or `q{ ... }` when strings don't have interpolated variables.

File: t/05register.t [hunk 2]
```diff
@@ -43,6 +43,18 @@ use DateTime::Locale;
...
+    qr/\Q'\@' or '=' are not allowed in locale ids\E/,
+    "locale id test with @"
```
Comment (autarch) [inline t/05register.t]: Same thing here.

**Review by autarch — CHANGES_REQUESTED** [review]
Looks good except for some small style changes. This would be another chance to practice rebasing ;)

**Comment by manwar** [comment]
Hi @autarch 

Is this acceptable? I have not commited the changes yet. If you are happy with the changes, may I put another commit with these changes and pushed the same branch?

```
  like(
  -    exception { DateTime::Locale->register( id => "@" ) },
  +    exception { DateTime::Locale->register( id => '@' ) },
         qr/\Q'\@' or '=' are not allowed in locale ids\E/,
  -    "locale id test with @"
  +    q{locale id test with @}
  );
```

> ``
> Many Thanks.
> Best Regards,
> Mohammad S Anwar

**Comment by autarch** [comment]
I'm not sure why you're usign `q{}` for the second string. Just use single quotes.

I was suggesting you simply commit on this same branch, then rebase to combine the two commits, then force push. That way you don't need to make a new PR.

**Comment by manwar** [comment]
Done as suggested. Please review again.

Done rebase for the first time without any issue ;-)

**Review by autarch — APPROVED** [review]

**Pull request merged**